src/extensions/default/CodeFolding/Prefs.js

```diff
@@ -21,9 +20,9 @@ define(function (require, exports, module) {
                            {name: strings.MIN_FOLD_SIZE, description: strings.MIN_FOLD_SIZE_HELP});
     prefs.definePreference("saveFoldStates", "boolean", true,
                            {name: strings.SAVE_FOLD_STATES, description: strings.SAVE_FOLD_STATES_HELP});
-    prefs.definePreference("alwaysUseIndentFold", "boolean", true,
+    prefs.definePreference("alwaysUseIndentFold", "boolean", false,
                            {name: strings.ALWAYS_USE_INDENT_FOLD, description: strings.ALWAYS_USE_INDENT_FOLD_HELP});
-    prefs.definePreference("enableRegionFolding", "boolean", true,
+    prefs.definePreference("enableRegionFolding", "boolean", false,
```

Comment:
It seems like the 'custom regions' stuff is partially removed, but not fully -- the menu item and the command behind it is gone, but this preference (and its strings) is still there along with region-fold.js.  Seems like we should remove those too (and then whatever extension re-exposes the menu item could reintroduce those elements too).  Does that sound ok?